ecms_base/config/install/user.role.anonymous.yml

```diff
@@ -16,4 +16,5 @@ permissions:
   - 'view paragraph content icon_card'
   - 'view paragraph content media_item'
   - 'view paragraph content promotion_reference'
+  - 'view paragraph content publication_list'
```

Comment:
I added this with RIG-178. A rebase should remove this.

Reply:
Also, is the 'card' paragraph new? We need to add the appropriate permissions to the anon/auth/author/publisher/admin users

Reply:
Think I got all of them now